core/src/main/scala/org/ensime/core/Analyzer.scala

```diff
@@ -51,7 +54,9 @@ class Analyzer(
   private var settings: Settings = _
   private var reporter: PresentationReporter = _
 
-  protected var scalaCompiler: RichCompilerControl = _
+  protected var scalaCompiler: Option[RichCompilerControl] = _
```

Comment:
yeah, this is a bit of a hack. We shouldn't need to do this, because the scala compiler should be on ENSIME's path not necessarily the project's. I'll try to send you a fix for this today.